hieradata/node/rucio.ls.lsst.org.yaml

```diff
@@ -0,0 +1,29 @@
+---
+nm::connections:
+  ens192:
+    content:
+      connection:
+        id: "ens192"
+        uuid: "810adb94-e2c0-4d41-b66f-1175a5c358ba"
+        type: "ethernet"
+        interface-name: "ens192"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.153.160/24,139.229.153.254"
```

Comment:
Why does this host need a static IP?

Reply:
Was no logic behind this, i assumed this host would require a private one. Configuration was updated for DHCP.